data/hai/hai reveal 1 intro.txt

```diff
@@ -11,47 +11,837 @@
+	npc accompany
+		government "Hai Merchant"
+		personality escort timid
+		system "Ula Mon"
+		fleet "Large Hai Merchant" 3
+	npc
+		government "Pirate"
+		personality coward opportunistic vindictive
+		system "Waypoint"
+		fleet "Marauder fleet IX" 2
+	on complete
+		"reputation: Hai" += 200
+		"reputation: Hai (Wormhole Access)" += 200
+		"reputation: Hai Merchant" += 200
+		"reputation: Hai Merchant (Human)" += 200
+		"reputation: Hai Merchant (Sympathizers)" += 200
+		event "hr: sandsnap awoken" 5
+		fail "HRCP: Sandsnap Mercenaries"
+		# Ideally eventually use: payment ( "armament deterrence" * 18000 - "cargo attractiveness" * 3000 )
+		payment 1200000
+		conversation
+			`As you're re-docking behind the supply convoy, a message pings on your console. It contains a short text from "Heyru" thanking you for your assistance and advising that you've been sent a payment of <payment> for your efforts.`
+			`	It seems that the security mechanisms in the docking section have been reactivated, so they no longer need their mercenaries to hang around.`
+
+
+event "hr: sandsnap awoken"
+	planet Sandsnap
+		government "Hai Merchant"
+		attributes hai station
+		landscape land/station35
+		music ambient/machinery
+		description `This station was recently re-powered by a private military contractor to serve as a resupply and repair base that Hai warships can be stationed at to respond more rapidly to pirates entering the system. There has been a constant presence here since it was set up shortly after the pirate raids on Allhome.`
+		spaceport `The operating model for the spaceport is evidently based on the military base at Everhope, but Sandsnap's layout makes restricting portions of the promenade unfeasible. While this leaves the dual-level promenade fully open to civilian access, security restrictions have been put in place at all access points to the higher levels.`
+		`	The floor has had new texturing laid down to prevent slipping, but nearly every other surface appears to be coated in a fine, sticky, metallic powder. Frequently touched surfaces have been polished clean, but otherwise any accidental contact with the powder results in fingerprints being left on any non-metallic surface until it is thoroughly wiped off.`
+		bribe 0.8
+		security 0.8
+
+
+mission "Hai Reveal Concern Period: Sandsnap Failed"
+	landing
+	invisible
+	source "Sandsnap"
+	to offer
+		has "Hai Reveal Concern Period: Securing Sandsnap: Failed"
+	on offer
+		log `Failed to escort the convoy safely to Sandsnap. The station will not be reactivated any time soon.`
+		conversation
+			`The Stationkeeper is distraught and palms you off to her subordinate who confirms that there is no payment he can give you.`
+			`	He explains that without the initial convoy arriving safely, there are now loans that were available contingent on success and that they simply don't have the money. It will take some time to raise money for a second attempt, if there is one.`
```

Comment:
```suggestion
			`	He explains that, without the initial convoy arriving safely, there are now loans that were available contingent on success and that they simply don't have the credits. It will take some time to raise the capital for a second attempt, if there is one.`
```